rust/pact_consumer/Cargo.toml

```diff
@@ -30,7 +30,7 @@ itertools = "0.12.1"
 lazy_static = "1.4.0"
 maplit = "1.0.2"
 pact_matching = { version = "~1.2.2", path = "../pact_matching", default-features = false }
-pact_mock_server = { version = "~1.2.6", default-features = false }
+pact_mock_server = { version = "=1.2.8", default-features = false, git = "https://github.com/you54f/pact-core-mock-server.git", branch = "main" }
```

Comment:
please update to released pact_mock_server, when pact_matching is released, and pact_mock_server updated to reference released crate